typeDefs.js

```diff
@@ -210,6 +214,8 @@ const typeDefs = gql`
     getUserByToken: User
     getUserById(id: ID!): User
 
+    validateToken(token: String!): LogicalResult
```

Comment:
Please, rename this query to the more concrete name because `validateToken` is too general and doesn't tell anything about the type of token that this operation checks

Reply:
fixed to validateConfirmationToken